packages/templates/clients/websocket/python/components/InitSignature.js

```
@@ -0,0 +1,24 @@
+      <Text indent={2} newLines={2}>
+        {`def __init__(self, url: str = "${serverUrl}"):`}
+      </Text>
+    );
+  }
+  
+  const queryParamsArguments = queryParams && queryParams.map((param) => {
+    const paramName = param[0];
+    const paramDefaultValue = param[1];
+    const defaultValue = paramDefaultValue ? `"${paramDefaultValue}"` : 'None';
+    return `${paramName}: str = ${defaultValue}`;
+  }).join(', ');
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)